### PR TITLE
Patch djangocms icons for d120

### PR DESCRIPTION
**Pull request by Xiphoseer**
This is a temporary fix to and config change for `djangocms-icon`to be able to use the fontawesome 4 icons that are already used by the website.

We likely need another fix to load the css in the admin views as well, kind of the inverse bug as we have now, where we can create font-awesome 5 icons because that is the default config and loaded from CDN in the admin but doesn't work in the frontend, because of the patched bug and the fact that we don't include the 5 icons at all.

### CODE REVIEW

**Comment by Xiphoseer** [comment]
CF https://github.com/django-cms/djangocms-bootstrap4/issues/124

**Comment by miterion** [comment]
I have added the website team since they are responsible for the maintenance of this project.

File: d120/templates/djangocms_bootstrap4/link/default/link.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    {% render_plugin plugin %}
+  {% empty %}
+    {{ instance.name }}
+  {% endfor %}
+
+  {% if instance.icon_right %}
+    <i class="{{ instance.icon_right }}"></i>
+  {% endif %}
+</a>
+{% endspaceless %}
```
Comment (bhaettasch) [inline d120/templates/djangocms_bootstrap4/link/default/link.html:26]: Blank line at end of file missing

**Review by bhaettasch — APPROVED** [review]
Looks fine to me, did not encounter any problems when testing locally.

File: d120/templates/djangocms_bootstrap4/link/default/link.html (hunk 1 above)
Comment (Xiphoseer) [reply d120/templates/djangocms_bootstrap4/link/default/link.html:26]: Das muss glaube ich so, wenn wir keinen Zeilenumbruch und damit Leerzeichen nach Buttons haben wollen

Comment (bhaettasch) [reply d120/templates/djangocms_bootstrap4/link/default/link.html:26]: OK :-)

**Pull request merged**